.github/workflows/reusable-deployment.yml

```diff
@@ -171,6 +171,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      - run: yarn cache clean
```

Comment:
so the reason you are having this issue is because you unplugged a couple of packages - that means you told yarn to not store them as zips inside `.yarn/cache` but rather unpack it and store in `.yarn/unplugged` and that folder (as opposed to the `cache` one) is not checked into git meaning that we have to do `yarn install` do get the contents of those unplugged packages. 

The way we go about those packages is that we run yarn install once and then cache it and then we use this step
```
       - name: Cache build typecheck output
        uses: ./.github/actions/cache-unplugged
        with:
          environment-name: ${{ inputs.environment-name }}
```

to load this cache into your current job. So just try and add this instead of cache clean and install.

Would be keen to understand why you had to unplug those packages?

Reply:
From what is written in prettier changelog I understood version 3 needs to be unplugged. Here in the description of this PR I've left the link to the changelog https://github.com/yldio/asap-hub/pull/4118#issue-2076592651